tests/filecheck/checks/check-line.test

```diff
@@ -0,0 +1,15 @@
+// RUN: sed -E "s/^(\/\/|;|#).*/ /" %s | grep -n foo | filecheck %s
```

Comment:
I don't think the `sed` script here is necessary, all we really want is to prefix all lines with their line-number, for which we can use something along the lines of `grep -n ^`

Reply:
Yeah, I was totally over-complicating that. It is still hard for me to wrap my head around what the test is and what the device under test is, because its all intertwined. :D

Fixed